### PR TITLE
Upgrade to serde 1.0

### PR DESCRIPTION
**Pull request by sb89**
#59

### CODE REVIEW

File: raw/src/types/update.rs [hunk 1]
```diff
@@ -48,9 +50,9 @@ impl Deserialize for Update {
         maybe_field!(edited_message, EditedMessage);
         maybe_field!(channel_post, ChannelPost);
         maybe_field!(edited_channel_post, EditedChannelPost);
-//        maybe_field!(inline_query, InlineQuery);
-//        maybe_field!(chosen_inline_result, ChosenInlineResult);
-//        maybe_field!(callback_query, CallbackQuery);
+        //        maybe_field!(inline_query, InlineQuery);
```
Comment (bjorn3) [inline raw/src/types/update.rs]: Please remove spaces after `//` except for one.

File: raw/src/types/update.rs [hunk 2]
```diff
@@ -48,9 +50,9 @@ impl Deserialize for Update {
...
+        //        maybe_field!(chosen_inline_result, ChosenInlineResult);
```
Comment (bjorn3) [inline raw/src/types/update.rs]: Same here

File: raw/src/types/update.rs [hunk 3]
```diff
@@ -48,9 +50,9 @@ impl Deserialize for Update {
...
+        //        maybe_field!(callback_query, CallbackQuery);
```
Comment (bjorn3) [inline raw/src/types/update.rs]: And here

File: raw/src/types/update.rs [hunk 4]
```diff
@@ -23,15 +23,17 @@ pub enum UpdateKind {
     ChannelPost(Message),
     /// New version of a channel post that is known to the bot and was edited
     EditedChannelPost(Message),
-//    InlineQuery(InlineQuery),
-//    ChosenInlineResult(ChosenInlineResult),
-//    CallbackQuery(OptionCallbackQuery),
+    //    InlineQuery(InlineQuery),
```
Comment (bjorn3) [inline raw/src/types/update.rs]: Also here

File: raw/src/types/message.rs [hunk 5]
```diff
@@ -313,7 +310,7 @@ pub struct RawMessage {
     pub audio: Option<Audio>,
     /// Message is a general file, information about the file.
     pub document: Option<Document>,
-//    pub game: Option<Game>,
+    //    pub game: Option<Game>,
```
Comment (bjorn3) [inline raw/src/types/message.rs]: And here

**Comment by bjorn3** [comment]
My review comments got out of order 😕

**Comment by knsd** [comment]
Thanks for the pr!

**Pull request merged**